docs/markdown/Using Pants/using-pants-in-ci.md

```diff
@@ -15,15 +15,46 @@ Directories to cache
 
 In your CI's config file, we recommend caching these directories:
 
-- `$HOME/.cache/pants/setup`: the initial bootstrapping of Pants.
-- `$HOME/.cache/pants/named_caches`: caches of tools like pip and PEX.
-- `$HOME/.cache/pants/lmdb_store`: cached content for prior Pants runs, e.g. prior test results.
+
+- `$HOME/.cache/pants/setup`<br>
+  This is the Pants bootstrap directory. Cache this against the version, as specified
+  in `pants.toml`.  E.g., if using GitHub Actions you can use:
+  ```yaml
+  path: |
+    ~/.cache/pants/setup
+  key: pants-setup-${{ runner.os }}-${{ hashFiles('pants.toml') }}
+  restore-keys: |
```

Comment:
This comment is left as a marker for all `restore-key` related comments which I won't repeat from https://github.com/pantsbuild/example-python/pull/104#discussion_r944978572

Reply:
(same comment about not being comfortable with using restore-keys)